addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -516,7 +516,9 @@ var SlideUploadDialog = Dialog.extend({
         if (!preventOnchange) {
             var input = file.name;
             var inputVal = input.substr(0, input.lastIndexOf('.')) || input;
-            this._formSetFieldValue('name', inputVal);
+            if(this._formGetFieldValue('name') === ""){
```

Comment:
Again linter issue, you should have some spacing in your if statement.

You should really make sure you have a correctly configured linter, it helps a lot.

Example of the output on my IDE:
![Selection_345](https://user-images.githubusercontent.com/41282536/71802853-643d8080-305f-11ea-97db-965bc28856de.png)